scripts/get_status.py

```diff
@@ -3,72 +3,102 @@
 import argparse
 import time
 import sys
-from pprint import pprint
+import pprint
 
 from gladier.utils.flow_generation import get_ordered_flow_states
-
-##import the client
 from gladier_xpcs.flow_online import XPCSOnlineFlow
+
+
 def arg_parse():
     parser = argparse.ArgumentParser()
     parser.add_argument("--run_id", help="The automate flow instance(run) to check.",
                         default=None)
     parser.add_argument("--step", help="The inside the flow execution to check",
                         default=None)
-    parser.add_argument("--timeout", help="How long to wait before exiting.",
-                        default=300)
+    parser.add_argument("--timeout", help="How long to wait before exiting. Defaults to "
+                                          "the original flow state or 300.",
+                        default=0)
```

Comment:
Shouldn't it default to 300?

Reply:
A bit of a hack. The default is 0 here so it can be de-prioritized under a user-value or the defined state value here: 
```
timeout = args.timeout or state_wait_time or 300
```